oauth2client/devshell.py

```diff
@@ -117,6 +124,9 @@ def __init__(self, user_agent=None):
     def _refresh(self, http_request):
         self.devshell_response = _SendRecv()
         self.access_token = self.devshell_response.access_token
+        if self.devshell_response.expires_in is not None:
+          self.token_expiry = (_UTCNOW() +
```

Comment:
4-space indent inside the `if` block. (PEP8)

I'd recommend changing this whole section as follows (to make it easier on the eyes):

```python
        expires_in = self.devshell_response.expires_in
        if expires_in is not None:
            delta = datetime.timedelta(seconds=expires_in)
            self.token_expiry = _UTCNOW() + delta
```